.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This project didn't come with one but `.vscode` would ordinarily be in a .gitignore file so these user-specific config files don't get committed.